bitbots_body_behavior/src/bitbots_body_behavior/decisions/goal_seen.py

```diff
@@ -9,6 +9,9 @@ def __init__(self, blackboard, dsd, parameters=None):
         self.goal_lost_time = self.blackboard.config['goal_lost_time']
 
     def perform(self, reevaluate=False):
+        self.publish_debug_data("goal_seen_time", rospy.get_time() - self.blackboard.world_model.goal_last_seen())
+        self.publish_debug_data("goal_x", self.blackboard.world_model.get_detection_based_goal_position_uv()[0])
```

Comment:
Since `get_detection_based_goal_position_uv´ is rather expensive due to TF I would suggest to call it only once (and evaluate if the call is even necessary for debugging purposes here or could be moved to another action where it is calculated anyway).